File: C#/Inserccion.cs
class Program
{

    static void Main()
    {
        Random x = new Random();
        int[] arr = new int[10];

        for (int i = 0; i < arr.Length; i++)
        {
            int numero = x.Next(1, 100);
            arr[i] = numero;
        }

        Console.WriteLine("El arreglo desordenado es: ");
        Array.ForEach(arr, x => Console.Write(x + " "));
        Console.WriteLine(" ");

        for (int i = 1; i < arr.Length; i ++)
        {
            int actual = arr[i];
            int j = i - 1;
            while (j >= 0 && arr[j] > actual)
            {
                arr[j+1] = arr[j];
                j = j - 1;
                
            }
            arr[j+1] = actual;

        }
        Console.WriteLine("El arreglo ordenado es: ");
        Array.ForEach(arr, x => Console.Write(x + " "));
    }


}

File: Python/Inserccion.py
import random
arr = []
for i in range(10):
    arr.append(random.randint(1,100))

print("El arreglo desordenado es: ", arr)

for i in range(1,len(arr)):
    actual = arr[i]
    j = i -1 
    while j >= 0 and arr[j] > actual:
        arr[j+1] = arr[j]
        j -= 1
    arr[j+1] = actual

print("El arreglo ordenado es: ", arr)